WordPress/src/test/java/org/wordpress/android/ui/mysite/items/listitem/SiteListItemBuilderTest.kt

```diff
@@ -417,18 +422,24 @@ class SiteListItemBuilderTest {
 
     /* SITE MONITORING */
     @Test
-    fun `give jetpack app, when is atomic and admin, then site monitoring item is built`() {
-        setupSiteMonitoringItems()
+    fun `give site monitoring FF is true, when is atomic and admin, then site monitoring item is built`() {
```

Comment:
@zwarm @AjeshRPai I modified a bit the unit tests since I transferred the check for the Jetpack app in the SiteMonitoringFeatureConfig. As I saw in the codebase, this is a common practice. If you don't agree with this, let me know to modify it again. :-)

Reply:
Thanks for the updates ❤️